symplyphysics/laws/conservation/abbe_invariant_of_two_optical_environments_is_constant.py

```diff
@@ -0,0 +1,41 @@
+from sympy import Eq, solve
+from symplyphysics import (Symbol, units, print_expression, Quantity,
+    validate_input, validate_output)
+
+# Description
+## The Abbe's invariant connects the front and back segments S and S',
+## allowing one of them to be determined if the second one is known
+
+# Law: Q = Q'
```

Comment:
Maybe it's better to combine both Abbe invariant definition and it's conservation to a single law? This way it is easier to understand, what's going on and no need to introduce Abbe invariant that does not have physical meaning.

Reply:
Initially, I also thought of drafting one law, but then I came across such a task:

"A thin plano-convex glass lens with a refractive index of n2 = 1.66, the convex side with a radius of curvature R = 12 cm is immersed in water. The refractive index of water is n1=1.33. At what distance from the lens and where will the image of the Sun be located at the zenith?"

![image](https://github.com/blackyblack/symplyphysics/assets/110999247/e847b0a5-b300-4b45-a88f-d51fd70557f3)

And I thought that if the beam passes through three media (for example, air, biconvex lens and water), then we will need an Abbe invariant for each medium, although I did not find such difficult tasks.

Reply:
I found an additional explanation:

"The Abbe formula is valid only when passing through one refractive surface, with a change in the position of the conjugate points, the value of the Abbe invariant will change, therefore it is not a complete invariant."

Therefore, yes, it is necessary to redo it into one law.